agent/src/adapter/agent.py

```
@@ -0,0 +1,513 @@
+                source, guard=self._guard, sink=sink, regenerate=regenerate
+            ):
+                if isinstance(out, str):
+                    spoke_anything = True
+                yield out
+        except (asyncio.CancelledError, GeneratorExit):
+            # Both derive from BaseException, so `except Exception` below would
+            # miss them anyway. Named explicitly because it is a decision, not
+            # an accident: barge-in and shutdown cancel this generator, that is
+            # not a failure, and speaking over it would be wrong.
+            raise
+        except Exception as exc:
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)